spring-cloud-app-starter-doc-maven-plugin/src/main/java/org/springframework/cloud/stream/app/documentation/plugin/ConfigurationMetadataDocumentationMojo.java

```diff
@@ -15,22 +15,11 @@
 
 package org.springframework.cloud.stream.app.documentation.plugin;
 
-
-import java.io.BufferedReader;
-import java.io.Closeable;
-import java.io.File;
-import java.io.FileReader;
-import java.io.IOException;
-import java.io.PrintWriter;
+import java.io.*;
```

Comment:
just curious why you use `*` for the imports since in other projects we explicitly import all the classes.

Reply:
This is certainly because I updated my IntelliJ installation and I understand nothing about how it's configured. Shouldn't the eclipse formatter plugin we use take care of that?

Reply:
Well, it may not work if IDEA's formatter is in auto mode.
See some discussion here: https://github.com/spring-cloud/spring-cloud-gcp/issues/101

Especially this my comment: https://github.com/spring-cloud/spring-cloud-gcp/pull/55#discussion_r121155979

Reply:
I was aware of that issue (sad that it doesn't work, btw) but it's not that.

Here is basically my config, maybe the eclipse files that are in skipper are wrong?

<img width="1465" alt="screen shot 2017-09-12 at 17 48 13" src="https://user-images.githubusercontent.com/313494/30335441-acbaa166-97e2-11e7-8416-798177b9a655.png">

Reply:
Do you know what "IJ's import optimizing must be turned on" preference refers to?

Reply:
Not sure, but it looks like something to do with the eclipse formatter.

Reply:
Yeah... Really not sure what is that, but my IDEA Auto Import looks like:
![image](https://user-images.githubusercontent.com/1071590/30336573-97fdeb40-97b3-11e7-8e98-ef798583de1f.png)

even if that  "IJ's import optimizing must be turned on" is turned on.

But at the same time I have this option in different place:
![image](https://user-images.githubusercontent.com/1071590/30336688-f0793e50-97b3-11e7-9062-2e024296e3bd.png)


Have just tested them with the `1` value and indeed all my imports moved to the asterisk style even with the Eclipse Formatter.

So, conclusion: the Eclipse Formatter is not a tool per se. And it depends on the many factors from the IDEA settings.

Reply:
Arghhh, great. So I guess I'll re-apply the IJ specific config file that we used to use.
Sorry about the diff noise